### ISSUE TITLE
Cleanup duplicate signups

### ISSUE THREAD

**Issue by itsjoekent** [DoSomethingArchive/legacy-website#6095]
- [ ] update `dosomething_signup_exists` to check for a signup without the run nid
- [ ] reconcile all duplicate signups
- [ ] reconcile all signups attached to the wrong run id

To find duplicate signups use this query

```
select s.nid, s.uid, s.run_nid, s.sid, from_unixtime(s.timestamp), r.field_current_run_target_id
from dosomething_signup s
inner join field_data_field_current_run r on r.entity_id = s.nid
where  1 < (select count(*) from dosomething_signup s2 where s2.uid = s.uid and s2.nid = s.nid);
```

This is blocking https://github.com/DoSomething/phoenix/issues/6014 as duplicates need to be removed before we can associate run nids on signups.

**Comment by deadlybutter** [comment]
questions @angaither 

1. after this is done, are we re-merging https://github.com/DoSomething/phoenix/issues/6020 ?
2. I'm assuming we're making an update hook for this, how are generating the new ID's to make them unique? Is there a drupal function for this?

**Comment by angaither** [comment]
closed with #6108
